upgrade-notes/backend_20240920_170120.md

```diff
@@ -0,0 +1,4 @@
+#### set the main blog category image as a background to the Magazine block on homepage ([#3450](https://github.com/shopsys/shopsys/pull/3450))
+
+-   graphql query `Settings` has now removed the `mainBlogCategoryUrl` field, use `mainBlogCategoryData` instead
```

Comment:
There has been also renaming of class and change of its constructor interface

Reply:
thanks for letting me know 👍, fixed in 405e5f64f4